src/main/resources/application-preprod.yaml

```
@@ -3,3 +3,5 @@ BA_SAK_API_URL: https://familie-ba-sak.dev-fss-pub.nais.io/api
 
 FAMILIE_BA_INFOTRYGD_SCOPE: api://dev-fss.teamfamilie.familie-ba-infotrygd/.default
 FAMILIE_BA_INFOTRYGD_API_URL: https://familie-ba-infotrygd.dev-fss.nais.io
```

```suggestion
FAMILIE_BA_INFOTRYGD_API_URL: https://familie-ba-infotrygd.dev-fss-pub.nais.io
```
Fikser I/O error `Remote host terminated the handshake `

---

Ok, thank you!